requirements.txt

```diff
@@ -1,38 +1,38 @@
 asgiref==3.7.2
-auth0-python==4.5.0
-beautifulsoup4==4.12.2
-boto3==1.26.143
-celery[sqs]==5.3.1
+auth0-python==4.7.0
+beautifulsoup4==4.12.3
+boto3==1.34.41
+celery[sqs]==5.3.6
 channels==4.0.0
 channels-redis==4.2.0
-daphne==4.0.0
-Django==4.2.10
+daphne==4.1.0
+Django==5.0.2
 django-crequest==2018.5.11
 django-extensions==3.2.3
-django-filter==22.1
+django-filter==23.5
 django-prometheus==2.3.1
 django-redis==5.4.0
-django-simple-history==3.3.0
-django-structlog==2.2.0
+django-simple-history==3.4.0
+django-structlog==7.1.0
 djangorestframework==3.14.0
 djproxy==2.3.6
 elasticsearch-dsl==7.4.1
-gunicorn==20.1.0
+gunicorn==21.2.0
 Jinja2==3.1.3
-kubernetes==25.3.0
-MarkupSafe==2.1.3
+kubernetes==29.0.0
```

Comment:
I'm a little bit unsure about this jump - there was some discussion with the rest of the team about whether we can upgrade when [dependabot suggested it](https://github.com/ministryofjustice/analytics-platform-control-panel/pull/1218). Probably not something that we would catch running locally